packages/synapse-react-client/src/components/SynapseTable/SynapseTable.tsx

```diff
@@ -843,48 +832,30 @@ export class SynapseTable extends React.Component<
   /**
    * Show the dropdown menu for a column that has been faceted
    *
-   * @param {number} index this is column index of the query table data
-   * @param {string} columnName this is the name of the column
-   * @param {FacetColumnResult[]} facetColumnResults
-   * @param {number} facetIndex
+   * @param {FacetColumnResult[]} facetColumnResult
+   * @param {ColumnModel} columnModel
    * @returns
    * @memberof SynapseTable
    */
   public configureFacetDropdown(
     facetColumnResult: FacetColumnResultValues,
     columnModel: ColumnModel,
-    lastQueryRequest: QueryBundleRequest,
   ) {
     return (
       <EnumFacetFilter
         containerAs="Dropdown"
         facetValues={facetColumnResult.facetValues}
         columnModel={columnModel}
-        onChange={facetNamesMap => {
-          applyMultipleChangesToValuesColumn(
-            lastQueryRequest,
-            facetColumnResult,
-            this.applyChangesFromQueryFilter,
-            facetNamesMap,
-          )
-        }}
-        onClear={() => {
-          applyChangesToValuesColumn(
-            lastQueryRequest,
-            facetColumnResult,
-            this.applyChangesFromQueryFilter,
-          )
-        }}
```

Comment:
EnumFacetFilter now directly uses the context instead of this complicated chain of callback props